kalite/distributed/features/steps/generic.py

```diff
@@ -1,6 +1,9 @@
 from behave import *
 from kalite.testing.behave_helpers import *
+from django.contrib.auth.models import User
 
 @given("I am on the homepage")
 def step_impl(context):
-    go_to_homepage(context)
\ No newline at end of file
+	if not User.objects.exists():
```

Comment:
> So we could say that tests that aren't supposed to care about superuser creation can get away with a simpler interface?

Maybe the compromise solution is to use tags in the `.feature` files (something like `@superuser_required`, and then add a snippet in [kalite.testing.base_environment.before_feature](https://github.com/learningequality/ka-lite/blob/develop/kalite/testing/base_environment.py#L26) to create a superuser if one doesn't exist.